lib/js/components/SelectList/SelectListItem/SelectListItem.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+	<div
+		v-ripple
+		class="selectListItem"
+		:class="{ '-loading': isLoading, '-selected': isSelected, [`-${size}`]: true }"
+		:title="label"
```

Comment:
(optional) when you disable `pointer-events` title is not dispalyed. Just informing, I won't force on fixing it

Reply:
We'd need a separate wrapper to fix it, right? Do you think it's worth it?

Reply:
probably not